src/Nox.EntityFramework.Postgres/PostgresUserDatabaseConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+using Nox.Types;
+using Nox.Types.EntityFramework.Types;
+
+namespace Nox.EntityFramework.Postgres;
+
+public class PostgresUserDatabaseConfiguration : UserDatabaseConfigurator
```

Comment:
Why do we need this configuration?

Postgres stores text data in text type, which is recommended anyways.

Reply:
We do not need this database-specific configuration.

I tested this with 3 different attributes
![image](https://github.com/NoxOrg/Nox.Generator/assets/131167535/f840d479-836a-41d4-b45d-22642b36069c)
which were created like this in the SqlServer database
![image](https://github.com/NoxOrg/Nox.Generator/assets/131167535/7ccd6827-eef2-4cbf-922b-dfd4b023264a)
and like this in the Postgres database
![image](https://github.com/NoxOrg/Nox.Generator/assets/131167535/9fdb8f7b-2a90-4ece-8bf2-17334127fda8)